### PR TITLE
[Code coverage] Cover the http status codes.

### PR DESCRIPTION
**Pull request by knightburton**
IoT.js-DCO-1.0-Signed-off-by: Imre Kiss kissi.szeged@partner.samsung.com

### CODE REVIEW

**Comment by yichoi** [comment]
@knightburton Please rebase it to the master and fix the travis issues

File: test/run_pass/test_net_http_status_codes.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  method: 'POST',
+  port: 3008,
+  headers: {'Content-Length': 3}
+};
+
+for (var i = 0; i < codes.length - 1; i++) {
+  var request = http.request(options, function(response) {
+    responses.push(response);
+    completedResponses++;
+    if (completedResponses == codes.length) {
```
Comment (hs0225) [inline test/run_pass/test_net_http_status_codes.js]: I doubt `if` statement will be executed. The maximum value of the `completedResponses` is `codes.length - 1`. And why do you request `http.request` as long as `codes.length - 1`, not `codes.length`?

**Comment by knightburton** [comment]
@hs0225 I've updated the PR, thank you for your review, you were right.

**Comment by yichoi** [comment]
LGTM

**Pull request merged**